### DISCUSSION TITLE
Is there a way to disable listing of deprecated linters in "Disabled by your configuration linters" using `golangci-lint linters` command?

### DISCUSSION THREAD

**Discussion by frederikhors** [golangci/golangci-lint#1970] [Q&A] [answered]
Is there a way to disable listing of deprecated linters in "Disabled by your configuration linters" using `golangci-lint linters` command?

**Comment by ldez** [comment, marked as answer]
There is no filter on decreprated linters, then you have to disable them explicitly:
- if you are using `--disable-all`, you just have to not enable them
- if you are using `--enable-all`, you have to add them to `--disable`
